packages/webpack-config/package.json

```
@@ -31,7 +31,12 @@
     "copy-webpack-plugin": "^5.0.0",
     "css-loader": "^2.1.1",
     "deep-diff": "^1.0.2",
+    "eslint": "^6.0.1",
     "eslint-loader": "^2.2.1",
+    "eslint-plugin-babel": "^5.3.0",
```

```suggestion
    "eslint-plugin-babel": "5.3.0",
```
Perhaps we should pin these versions? I think ESLint & plugins break very easily when new versions of them get published that can require upgrading other plugins or ESLint itself, or require some minor changes to the configuration. If we pin the version we can upgrade them ourself and test before publishing.